stellarphot/settings/models.py

```diff
@@ -80,3 +89,107 @@ class PhotometryFileSettings(BaseModel):
     aperture_locations_file: Path = Field(
         filter_pattern=["*.ecsv", "*.csv"], default=""
     )
+
+
+class TimeType(Time):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return Time(v)
+
+
+class SkyCoordType(SkyCoord):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return SkyCoord(v)
+
+
+class Exoplanet(BaseModel):
+    """
+    Create an object representing an Exoplanet.
+
+    Parameters
+    ----------
+
+    epoch : TimeType
+        Epoch of the exoplanet as an astropy Time object, not required.
+
+    period : QuantityType
+        Period of the exoplanet as a Quantity with units of time, not required.
+
+    Identifier : Str
```

Comment:
```suggestion
    Identifier : str
```

Python types are lower case.